### PR TITLE
HoG Update

### LINKED ISSUE #12980: HoG objectives are bugged
shtuff

Nothing objective

### PR DESCRIPTION
**Pull request by RemieRichards**
## Changes:
- Deities now start off with the ability to place 1 free turret
- HoG Claymores are now a subtype, Damage: 40 --> 30, Armour Penetration: 0 --> 15, this reduces their ability to kill innocents, but keeps their ability to kill sec/the other cult about the same
- Deicide objective (Kill the other god!) will now correctly find the blue god
- Turrets will no longer attack handcuffed people, this should help with conversion, you can still turn turrets on/off.
- Followers are now informed of their god's death
- Followers are now informed of the location of their god's nexus when it is placed
- Conduit/Nexus range calculations are now cheaper
- Gods can no longer place structures near the enemy god's nexus, while hilarious, this cant happen anymore:
  hilarious
## Fixes:

Fixes #12980 

:cl:
add: Deities now start off with the ability to place 1 free turret
tweak: HoG Claymores now do 30 Brute (Down from 40) but now have 15 Armour Penetration (Up from 0)
fix: Killing the other god objective is fixed
tweak: Turrets will no longer attack handcuffed people
add: Followers are now informed of their god's death
add: Followers are now informed of the location of their god's nexus when it is placed
tweak: Gods can no longer place structures near the enemy god's nexus
/:cl:

### CODE REVIEW

**Comment by tkdrg** [comment]
> HoG tweaks have a changelog
> HoG itself has no changelog

That aside, :+1:, though I would rephrase the turret change as "Turrets will no longer _target_ handcuffed people", since if you're standing between a turret and its target you are still going to get shot.

File: code/game/gamemodes/handofgod/god.dm [hunk 1]
```diff
@@ -97,17 +120,31 @@
 	//verbs += /mob/camera/god/verb/movenexus //Translocators have no sprite
 	update_health_hud()
 
+	var/area/A = get_area(src)
+	if(A)
+		var/areaname = A.name
+		var/list/followers = get_my_followers()
+		for(var/datum/mind/F in followers)
+			if(F.current)
+				F.current << "<span class='boldnotice'>Your god's nexus is in [areaname]</span>"
```
Comment (tkdrg) [inline code/game/gamemodes/handofgod/god.dm]: `\the [areaname]`

**Comment by Aloraydrel** [comment]
Lookin good :+1:
